omod/src/main/webapp/resources/htmlforms/055b-HCTClientCard.xml

```diff
@@ -0,0 +1,596 @@
+                        </td>
+                        <td colspan="3">
+                            <obs conceptId="99463"/>
+                        </td>
+                    </tr>
+
+
+                    <tr>
+                        <td>
+                            <label>Your spouse/partner ever been tested for HIV before :</label>
+                        </td>
+                        <td>
+                            <!--                                 not applicable answer among the answers is missing-->
+                            <obs id="99472" conceptId="99472" answerConceptIds="1065,1066,99480,90001" answerLabels="1 - Yes,2 - No,3 - Don't Know, 4 - Not Applicable" />
+
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <label style="float:right"><i>If yes, the most recent test results for the spouse</i></label>
+                        </td>
+                        <td class="PartnerTestedBeforeEnableDisableChild1">
+                            <obs id="99477" conceptId="99477" answerConceptIds="90166,90167"
+                                 answerLabels="1 - Positive ,2 - Negative ,3 - Don't know" style="radio"/>
+                        </td>
+                    </tr>
+                </excludeIf>
+
+
+                <tr>
+                    <td style="width: 20%">
+                        <label>Pre -Test Counseling done/information given:</label>
+                    </td>
+                    <td>
+                        <obs id="162918" conceptId="162918" answerConceptIds="90003,90004" answerLabels="1 - Yes, 2 - No" style="radio"/>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <label>Counseled as:</label>
+                    </td>
+
+                    <td colspan="3" class="PreTestCounselingEnableDisableChild1">
+                            <span class="CounseledAsCoupleEnableDisable1">
+                                <obs conceptId="99368" id="99368" answerConceptIds="99365,99367,99366"
+                                     answerLabels="1 - Individual,2 - Couple,3 - Group" style="radio"/>
+                            </span>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+        </section>
+
+        <section>
+            <h3>SECTION D:HIV TEST RESULTS</h3>
+            <table class="section-b">
+                <tbody>
+                <tr>
+                    <td style="width: 20%"><label>HIV final test results:</label>
+                    </td>
+                    <td>
+                        <obs id="99493" conceptId="99493" answerConceptIds="90167,90166,162926" answerLabels="1 - Negative,2 - Positive,3 - Inconclusive" style="radio" required="true"/>
+                    </td>
+                </tr>
+
+                <tr>
```

Comment:
@dbaluku Move this to be after Syphilis Duo

Reply:
Done